com.github.rafostar.Clapper.json

```diff
@@ -0,0 +1,58 @@
+{
+    "app-id": "com.github.rafostar.Clapper",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.38",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.rafostar.Clapper",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=all",
+        "--filesystem=xdg-videos",
+        "--talk-name=org.gnome.Shell",
```

Comment:
What is it needed for?

Reply:
Wayland doesn't have any protocol to show PiP video always on top, so ask GNOME Shell instead. Having floating PiP video is probably expected by majority of users. It is also used to optionally show that video on all workspaces if user wishes to.